dynamiqs/plots/misc.py

```diff
@@ -161,6 +170,98 @@ def plot_wigner(
     )
 
 
+def plot_wigner_mosaic(
+    states: ArrayLike,
+    *,
+    n: int = 8,
+    ncols: int | None = None,
```

Comment:
I think `nrows` would be more natural. Because if not specified, then `nrows = 1`.

Reply:
Right 👍